README.md

```diff
@@ -1,13 +1,14 @@
-# Jenkins UI sample plugin
+![Jenkins Design Library](logo-dark.svg#gh-dark-mode-only)
+![Jenkins Design Library](logo.svg#gh-light-mode-only)
 
-This fake plugin shows Jenkins developers how to build user interfaces using Jenkins UI components.
+Design Library makes it easy for developers to build complex and consistent interfaces using Jenkins UI components
 
 This plugin contains:
-- demonstration of UI controls available in Jenkins based on Stapler, Jelly, Groovy, etc.
-- snippets of Jelly/Groovy "ready-to-use" examples
+- demonstration of UI components available in Jenkins
+- snippets of Jelly ready-to-use examples
 
 ## Usage
```

Comment:
We could add a section with additional information like linking to the jelly tag reference used in core and a link to the gitter room for ux-sig where people could head to for additional information.

Reply:
I think that should belong in the contributing document see https://github.com/jenkinsci/ui-samples-plugin/issues/16

Reply:
Oic, I didn't see your issue here.